invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -264,6 +264,19 @@ def get_dates(self, obj):
 
             dates.append(to_append)
 
+        if not updated:
+            try:
+                udate = obj["updated"]
```

Comment:
```suggestion
                updated_date = obj["updated"]
```
not a blocker: for readability reasons could you use the full name for the variable? we had a problem with understanding it

Reply:
Thanks! I've updated and rebased